modules/core/src/main/scala/org/scalasteward/core/nurture/NurtureAlg.scala

```diff
@@ -230,20 +230,18 @@ final class NurtureAlg[F[_]](config: ForgeCfg)(implicit
           .on(u => List(u.currentVersion.value), _.updates.map(_.currentVersion.value))
           .flatTraverse(gitAlg.findFilesContaining(data.repo, _))
           .map(_.distinct)
-      branchName = forge.createBranch(config.tpe, data.fork, data.updateBranch)
       allLabels = labelsFor(data.update, edits, filesWithOldVersion, artifactIdToVersionScheme)
       labels = filterLabels(allLabels, data.repoData.config.pullRequests.includeMatchedLabels)
-      requestData = NewPullRequestData.from(
-        data = data,
-        branchName = branchName,
-        edits = edits,
-        artifactIdToUrl = artifactIdToUrl,
-        artifactIdToUpdateInfoUrls = artifactIdToUpdateInfoUrls.toMap,
-        filesWithOldVersion = filesWithOldVersion,
-        addLabels = config.addLabels,
-        labels = data.repoData.config.pullRequests.customLabels ++ labels
-      )
-    } yield requestData
+    } yield NewPullRequestData.from(
+      data = data,
+      branchName = config.tpe.pullRequestHeadFor(data.fork, data.updateBranch),
+      edits = edits,
+      artifactIdToUrl = artifactIdToUrl,
+      artifactIdToUpdateInfoUrls = artifactIdToUpdateInfoUrls.toMap,
+      filesWithOldVersion = filesWithOldVersion,
+      addLabels = config.addLabels,
+      labels = data.repoData.config.pullRequests.customLabels ++ labels
+    )
```

Comment:
This is a small clean-up I couldn't resist-  there's no point in defining `branchName` & `requestData` back on old lines 233 & 236 - they're just used again immediately just once, and in both cases there's already sufficient context in the code for us to well understand what they are (`NewPullRequestData`'s named parameter `branchName`, and then the name of the `NewPullRequestData` type itself).